visualize-admin/visualization-tool
# feat: Standard errors in line charts

Closes #1501

This PR adds support for displaying standard errors in line charts and updates the design of error whiskers along with configurator panel options for standard errors.

## How to test
1. Go to [this link](https://visualization-tool-git-feat-standard-errors-in-line-charts-ixt1.vercel.app/en/create/new?cube=https://environment.ld.admin.ch/foen/emis/yearly/cube/2023/&dataSource=Test).
3. Switch to a line chart.
4. See that the error whiskers are displayed.
5. Open a Vertical Axis panel.
6. See that there's a new design for enabling standard errors (switch element with a tooltip explaining the behavior).
7. Hover over the chart and see that information about errors is displayed in the tooltip.
